lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -162,9 +164,55 @@ define((require, exports, module) ->
     ch = ContextHandler = {
       contextPreviewExists: false
       UPDATE_DELAY: 1000
+      LABEL_SEQUENCE: [
+        {type: "keyword", value: /^\\label$/}
+        {type: "lparen", value: /^\{$/}
+        {type: "variable.parameter", value: /.*/}
```

Comment:
is it true that all text inside label is variable.parameter even if it is actually another macro with its own name, brackets and arguments?